src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/FireUtilitySpell.java

```diff
@@ -64,10 +65,12 @@ public static void onPlayerTick(TickEvent.PlayerTickEvent event) {
             Player player = event.player;
             List<FireUtilitySpell> equippedSpells = getEquippedSpellItemsOfType(FireUtilitySpell.class, player);
             if (equippedSpells.size() > 0) {
+                FireUtilitySpell highestSpell = getHighestSpellItem(equippedSpells);
+                player.getCapability(PlayerManaProvider.PLAYER_MANA).ifPresent(playerMana -> playerMana.setManaCapModifier("fire_utility", -highestSpell.manaReduction));
+                player.addEffect(new MobEffectInstance(MobEffects.FIRE_RESISTANCE, 2, 1, false, false));
```

Comment:
Set the `amplifier` to zero since effect levels start at 0.

Reply:
fixed